db/migrate/20200114164456_normalize_ace_table.rb

```diff
@@ -0,0 +1,33 @@
+class NormalizeAceTable < ActiveRecord::Migration[5.2]
+  def up
+    create_table :permissions do |t|
+      t.integer :name
+
+      t.timestamps
+    end
+
+    create_join_table :access_control_entries, :permissions do |t|
+      t.index [:access_control_entry_id, :permission_id], :name => "index_ace_permissions_on_ace_id_and_permission_id"
+    end
+
+    (0..3).each do |permission|
```

Comment:
@syncrou What's this doing its creating permission with names of 0,1,2,3? Shouldn't these be create, update, read, delete, order etc.